cluster/job_once_example_test.go

```diff
@@ -0,0 +1,40 @@
+package cluster
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+func HandleJobOnceCalls(key string) {
+	if key == "the key i'm watching for" {
+		// Work to do only once per cluster
+	}
+}
+
+func ExampleScheduleOnce() {
+	// Use p.API from your plugin instead.
+	pluginAPI := plugin.API(nil)
+
+	// Get the scheduler, which you can pass throughout the plugin...
+	scheduler := GetJobOnceScheduler(pluginAPI)
+
+	// Set the plugin's callback handler
+	_ = scheduler.SetCallback(HandleJobOnceCalls)
+
+	// Now start the scheduler, which starts the poller and schedules all waiting jobs.
+	_ = scheduler.Start()
+
+	// main thread...
+
+	// add a job
+	_, _ = scheduler.ScheduleOnce("the key i'm watching for", time.Now().Add(2*time.Hour))
+
+	// Maybe you want to check the scheduled jobs, or close them:
+	jobs, _ := scheduler.ListScheduledJobs()
+	defer func() {
+		for _, j := range jobs {
+			scheduler.Close(j.Key)
```

Comment:
Nope -- maybe I should remove that or add a comment, closing is totally optional. "Close" means stop a currently scheduled job so that it will never be run, on this instance or any other instance.

Reply:
Based on @agarciamontoro  comment I'll change that to `scheduler.Cancel(j.key)` to be clearer.